### PR TITLE
STRAT 99 - Update auxiliary loss power budget - Updated Test

### PR DESCRIPTION
**Pull request by SallyYang02**
Changed values from test based on asserting errors.

### CODE REVIEW

**Review by e-wai — CHANGES_REQUESTED** [review]
You're missing the new power budget file! (You'll need to `git add "MSXIV LV Power Budget Rev 2.csv"` and then commit again)

**Comment by SallyYang02** [comment]
> You're missing the new power budget file! (You'll need to `git add "MSXIV LV Power Budget Rev 2.csv"` and then commit again)

Thanks! Just realized that. Did another pull request with the updated power budget file. Since I don't have permission to add it directly, would you be able to approve the PR?

File: auxloss/tests/test_aux_power_consumption.py [hunk 1]
```diff
@@ -38,25 +38,25 @@ def test_expected_headers_stored():
         assert("Max Power (W)" in headers and "Typical Power (W)" in headers)
 
 def test_calculate_instantaneous_power_string_names_only():
-    assert(auxpc.calculate_instantaneous_power({"components": ['Center Console']}) == 4.4)
-    assert(auxpc.calculate_instantaneous_power({"components": ["Horn", "Motor Interface", "Solar Master"]}) == 0 + 2.4 + 4.0)
+    assert(auxpc.calculate_instantaneous_power({"components": ['Center Console']}) == 16.54)
```
Comment (e-wai) [inline auxloss/tests/test_aux_power_consumption.py:41]: I think it's worth identifying here which components are unknowns according to the new budget and removing them from the upper tests. That logic is tested in the `test_calculate_instantaneous_power_ignores_unknown_components`. We can replace them with new components in the power budget instead and update the expected values 👍

**Review by e-wai — CHANGES_REQUESTED** [review]

**Comment by e-wai** [comment]
Because we messed with the git history, you might have to reset your local branch:
- `git checkout update-auxiliary-loss-power-budget`
- `git fetch`
- `git reset --hard origin/update-auxiliary-loss-power-budget`

I've also noticed two different files between @SallyYang02 and @AnhMai-bit => `MSXIV LV Power Budget Rev 2 - Summary.csv` vs. `MSXIV LV Power Budget Rev 2.csv`. Maybe check that out? This version uses the first one

File: auxloss/tests/test_aux_power_consumption.py (hunk 1 above)
Comment (AnhMai-bit) [reply auxloss/tests/test_aux_power_consumption.py:41]: Hi Emma, can you explain how to approach this? We are not sure of what to do from the comment.

**Review by e-wai — APPROVED** [review]
Looks good to me! 🎉

**Comment by e-wai** [comment]
Did you guys want to merge it or should I?

**Pull request merged**